CONTRIBUTING.md

```diff
@@ -1,15 +1,78 @@
-## Welcome!
+## Getting Started
 
-We're so glad you're thinking about contributing to an 18F open source project! If you're unsure or afraid of anything, just ask or submit the issue or pull request anyways. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contribution, and don't want a wall of rules to get in the way of that.
+### Running the app locally
 
-Before contributing, we encourage you to read our CONTRIBUTING policy (you are here), our LICENSE, and our README, all of which should be in this repository. If you have any questions, or want to read more about our underlying policies, you can consult the 18F Open Source Policy GitHub repository at https://github.com/18f/open-source-policy, or just shoot us an email/official government letterhead note to [18f@gsa.gov](mailto:18f@gsa.gov).
```

Comment:
I believe the license stuff belongs in CONTRIBUTING.md by convention.

As referenced here, https://github.com/18F/open-source-policy/blob/master/README.md, README.md typically has a couple lines about license/public domain so that it is clear on the github home page.